tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -291,6 +312,13 @@ def get_auth_options(self, access_token, sub):
         auth_options = {"login_gov_uuid": sub}
         return auth_options
 
+    def handle_email(self, user):
+        """Handle user email exception to disallow ACF staff to utilize non-AMS authentication."""
```

Comment:
it looks like this is handling the scenario where an ACF user who was approved and assigned a role authenticates via login.gov at next sign-in. 👍🏾 how are we handling the first-time user request where they authenticate via login.gov?

Reply:
This function is unaware if it is a first-time user or not. Because this is called _after_ all the logic in `handle_user()` we'll let the user get created but then fail the authentication process as a whole.